storage/store.go

```diff
@@ -1452,9 +1452,51 @@ func (s *store) canUseShifting(uidmap, gidmap []idtools.IDMap) bool {
 // On entry:
 // - rlstore must be locked for writing
 // - rlstores MUST NOT be locked
```

Comment:
This is not true any more … in fact we do want the store containing the parent, if any, to be locked. See elsewhere about inlining instead.